packages/client/tests/functional/_example/tests.ts

```diff
@@ -68,4 +68,10 @@ testMatrix.setupTestSuite(
   //     reason: 'Only testing xyz provider(s) so opting out of sqlite and mongodb',
   //   },
   // },
+  {
+    skipEngine: {
+      from: ['wasm'],
+      reason: 'Fails on init with `unwrap_throw` failed',
+    },
+  },
```

Comment:
Trying this locally, this can be unskipped 

```
 PASS  tests/functional/typescript/tests.ts (5.624 s)
  typescript
    ✓ _example (provider=sqlite, previewFeatures=referentialIntegrity, js_libsql)/tests.ts (1 ms)
    ✓ _example (provider=sqlite, previewFeatures=tracing, js_libsql)/tests.ts
    ✓ globals.d.ts
```